client/src/services/psi-cohort.js

```diff
@@ -1,4 +1,5 @@
 import store from 'store';
+import moment from 'moment';
```

Comment:
might want dayjs here instead?

Reply:
Currently front end is using moment and backend dayjs, one of the first thing in my tech debt backlog is use dayjs for front end